eahub/templates/components/feedback.html

```diff
@@ -0,0 +1,144 @@
+{% block feedback %}
+
+<style>
+  @media (min-width: 1200px) {
+  #feedback {
+    right: -420px;
+    width: 420px;
+    }
+  }
+  
+  @media (max-width: 1200px) {
+  #feedback {
+    right: -200px;
+    width: 200px;
+  }
+
+  #closebtn {
+    display: none;
+  }
+  }
+
+  /*--- All styles ---*/
+
+  #feedback {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    min-height: 450px;
+    position: fixed;
+    top: 50%;
+    transform: translateY(-50%);
+    transition: right 0.5s linear;
+    padding: 0 58px 58px;
+    z-index: 1090;
+  }
+
+  #feedback.active {
+      right: 0;
+      box-shadow: 0 10px 18px rgba(0,0,0,0.16);
+  }
+  
+  .modal-content {
+    height: 100%;
+  }
+  
+  #overlay {
+    background-color: rgba(0,0,0,0.4);
+      position: fixed;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      z-index: 1080;
+      opacity: 0;
+      visibility: hidden;
+      transition: all 0.5s linear;
+  }
+  
+  #overlay.active {
+    visibility: visible;
+    opacity: 1;
+  }
+  
+  #feedbackModal{
+    display: block;
+  }
+
+  .feedbackbtn {
+    border: 0;
+    color: #fff;
+    background: rgb(99, 82, 116);
+    border-color: rgb(99, 82, 116);
+    cursor: pointer;
+    line-height: 44px;
+    left: 0;
+    padding: 0 28px;
+    position: absolute;
+    text-align: center;
+    top: 50%;
+    transform: rotate(-90deg) translate(-50%, -100%);
+    transform-origin: top left;
+    box-shadow: 0 0 0 rgba(0,112,153,0.4);
+  }
+
+  #closebtn {
+    color: #fff;
+  }
+
+  .submitbtn {
+    color: #fff;
+    background: rgb(99, 82, 116);
+    border-color: rgb(99, 82, 116);
+  }
+</style>
+  
+  <div id="overlay"></div>
+
+  <div id="feedback">
+  <!-- Button trigger modal -->
+  <button type="button" class="btn feedbackbtn" onClick="togglemodal()">
```

Comment:
a correction: in bs5 it should be `data-bs-target`, more details can be found here - https://getbootstrap.com/docs/5.0/components/modal/

FYI `onclick` isn't used those days, it's left for the old browsers sake, because there are scripts that silently override it and drop the togglemodal() call :)